app/Atro/Migrations/V1Dot8Dot41.php

```
@@ -0,0 +1,282 @@
+        } catch (\Throwable $e) {
+
+        }
+    }
+
+
+    protected function createIntermediateTable(): void
+    {
+       if($this->isPgSQL()){
+            $this->execute("CREATE TABLE extensible_enum_extensible_enum_option (id VARCHAR(24) NOT NULL, deleted BOOLEAN DEFAULT 'false', created_at TIMESTAMP(0) WITHOUT TIME ZONE DEFAULT NULL, modified_at TIMESTAMP(0) WITHOUT TIME ZONE DEFAULT NULL, sorting INT DEFAULT NULL, created_by_id VARCHAR(24) DEFAULT NULL, modified_by_id VARCHAR(24) DEFAULT NULL, extensible_enum_id VARCHAR(24) DEFAULT NULL, extensible_enum_option_id VARCHAR(24) DEFAULT NULL, PRIMARY KEY(id));");
+            $this->execute("CREATE UNIQUE INDEX IDX_EXTENSIBLE_ENUM_EXTENSIBLE_ENUM_OPTION_UNIQUE_RELATION ON extensible_enum_extensible_enum_option (deleted, extensible_enum_id, extensible_enum_option_id);");
+            $this->execute("CREATE INDEX IDX_EXTENSIBLE_ENUM_EXTENSIBLE_ENUM_OPTION_CREATED_BY_ID ON extensible_enum_extensible_enum_option (created_by_id, deleted);");
+            $this->execute("CREATE INDEX IDX_EXTENSIBLE_ENUM_EXTENSIBLE_ENUM_OPTION_MODIFIED_BY_ID ON extensible_enum_extensible_enum_option (modified_by_id, deleted);");
+            $this->execute("CREATE INDEX IDX_50529C766F0CDC2EF0427B90BDB97B1C ON extensible_enum_extensible_enum_option (extensible_enum_id, deleted);");
+            $this->execute("CREATE INDEX IDX_E67BAD4E2BB69A9BBC26754190C14C9D ON extensible_enum_extensible_enum_option (extensible_enum_option_id, deleted);");
+            $this->execute("CREATE INDEX IDX_EXTENSIBLE_ENUM_EXTENSIBLE_ENUM_OPTION_CREATED_AT ON extensible_enum_extensible_enum_option (created_at, deleted);");
+            $this->execute("CREATE INDEX IDX_EXTENSIBLE_ENUM_EXTENSIBLE_ENUM_OPTION_MODIFIED_AT ON extensible_enum_extensible_enum_option (modified_at, deleted);");
+
+            if($this->getCurrentSchema()->getTable('extensible_enum_option')->hasIndex('idx_extensible_enum_option_unique_option')){
```

No needs to check it here because you ignore any errors